pkg/edgeview/Makefile

```diff
@@ -0,0 +1,13 @@
+#
+# example to build a docker image for edge-view, for query/client usage
+# github build process may want to push the image into certain docker repository
+#
+edge-view-query:
+	docker build -f Dockerfile.query -t edge-view-query .
+
+#
+# build the websocket server/dispatcher, should compile on a machine in the
+# same architecture as the server runs the websocket dispather
+#
+wss-server:
```

Comment:
Wait, so we really need 3 containers here correct?
  * wss-server
  * client
  * the one that gets deployed on EVE

Reply:
@rvs for EVE device, only the last one is needed. but the end-to-end service (besides the controller UI piece) the client and wss-server are needed, here I just offer a way to compile them and to use the same repo

Reply:
@naiming-zededa when we build the one which gets deployed on EVE I assume it is built for Alpine i.e., busybox environment. Are the other two built as native for the built system? For instance, can one build client for Ubuntu and for Windows?

Reply:
@eriknordmark the EVE and 'client' are basically the same docker image both with
- FROM lfedge/eve-alpine:6.7.0 as build

the 'dispatcher' is an example code there, currently don't have docker wrapper around, i just build on an ubuntu machine and copy the binary to the dispatcher server (so it does not need to have all the docker infra).